go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/linkerd/linkerd-smi
+
+go 1.16
+
+require (
+	github.com/elazarl/goproxy/ext v0.0.0-20210110162100-a92cc753f88e // indirect
+	github.com/linkerd/linkerd2 v0.5.1-0.20210518142200-847b7441831a
```

Comment:
That's the latest version. Currently the linkerd pkgs are not versioned and published to go pkgs i.e https://pkg.go.dev/github.com/linkerd/linkerd2. I think, the reason is because we don't use semver.

We instead directly depend on a specific commit, which go mod for some reason weirdly parsed as `v0.5.1` 🤔